mmhuman3d/data/datasets/human_image_dataset.py

```diff
@@ -45,6 +51,13 @@ class HumanImageDataset(BaseDataset, metaclass=ABCMeta):
         convention (str, optional): keypoints convention. Keypoints will be
             converted from "human_data" to the given one.
             Default: "human_data"
+        cache_data_path (str | None, optional): the path to store cache files.
+            When cache_data_path is None, each dataset will store a copy into
+            memory. If cache_data_path is set, the dataset will first create
+            cache files and then use a cache reader to reduce memory cost and
```

Comment:
`cache files` to `one cache file`? It refers to one file in type 'npz', but the docstring here misses the infomation.

Reply:
fixed